packages/@adobe/spectrum-css-temp/components/dropzone/skin.css

```diff
@@ -18,15 +18,33 @@ governing permissions and limitations under the License.
 .spectrum-Dropzone {
   border-color: var(--spectrum-dropzone-border-color);
 
-  &.is-dragged {
+  &[data-focus-visible]{
+    border-color: var(--spectrum-dropzone-border-color-selected-hover);
+
+    svg {
+      color: var(--spectrum-dropzone-illustratedmessage-illustration-color-key-focus);
+    }
+  }
+
+  &[data-focused] {
+    border-color: var(--spectrum-dropzone-border-color-selected-hover);
+  }
```

Comment:
i'm not too sure either. it hasn't been discussed with design so definitely worth bringing it up to them. for now, i think i'll have it match the focus-visible style and we can change it later according to what design says

Reply:
I would lean towards not having the `data-focused` style, since it doesn't point the user towards any particular action they could take.

Reply:
well...technically they could paste but that's really the only action i can think of. i think it would be a bit weird if you were to click on the dropzone to paste something and nothing changed visually but also maybe it's worth reconsidering if that's an action we want to allow?

Reply:
Shouldn't this type of selection/focus be done with the FocusRing component?

Reply:
hmm well this styling really is only applied when you click on the dropzone with a mouse and i believe that the FocusRing component only applies for keyboard interactions so im not sure if that would fix things but let me know if im missing anything